commitlint.config.js

```diff
@@ -0,0 +1,51 @@
+// commitlint.config.js
+module.exports = {
+    rules: {
+        // log-commit-info is used for debugging. It outputs the dict passed to a plugin.
+        // 'log-commit-info': [2, 'always'],
+        'breaking-change-indicator': [2, 'always'],
```

Comment:
We can explain for better readability in one comment: [2 - consider as an error, 'always' - rule always applicable]

Reply:
done. also linked to the documentation where this is described.